Backend/Remora.Discord.Rest/API/Guilds/DiscordRestGuildAPI.cs

```diff
@@ -533,6 +533,26 @@ public virtual Task<Result> ModifyGuildMemberAsync
         }
 
         /// <inheritdoc />
+        public virtual Task<Result<IGuildMember>> ModifyCurrentMemberAsync
```

Comment:
Remember to add an override in the caching library as well :)

Reply:
Ah, sure thing!